docs/tutorial.md

```diff
@@ -255,8 +255,8 @@ spec:
           name: skaffold-image-leeroy-web
 ```
 
-To apply the yaml files use the following command, you need to apply the two
-`PipelineResources`, the `Task` and `TaskRun`.
+To apply the YAML files use the following command, you need to apply the two
```

Comment:
"using"? This sentence is grammatically confusing. You're either trying to say:

"To apply the YAML files using the following command, you need to..." OR

"To apply the YAML files, use the following command. You need to..." 

Which one is it?